.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: weekly
+    ignore:
+      # Breaks backwards compatibility
+      - dependency-name: github.com/gravitational/ttlmap
+      # Must be kept in-sync with libbpf
+      - dependency-name: github.com/aquasecurity/libbpfgo
+      # Forked/replaced dependencies
```

Comment:
Would it make sense to add a quick note to our go.mod's `replace ( ... )` with a reminder to sync fork updates back here, or will these be reasonably stable?

Reply:
Good call, done.